src/Elmish/Browser.purs

```diff
@@ -0,0 +1,37 @@
+module Elmish.Browser
```

Comment:
I don't think this is a good name for the module. It makes it look like the rest of the library is somehow intended to work outside the browser, and this one module brings the browser in.

Reply:
Yeah, I can see that. This came from https://package.elm-lang.org/packages/elm/browser/latest/Browser#sandbox. I figure this being Elmish we should first try and pick a name from either Elm or React (`render`) instead of inventing a new one. `sandbox` isn’t great because we don’t have the equivalent of subscriptions. In that sense, maybe `Elm.Browser.element` might be a good fit given our approach doesn’t give control over `<title>` and `<body>`.